cypress/e2e/po/pages/page.po.ts

```diff
@@ -9,6 +9,14 @@ export default class PagePo extends ComponentPo {
     return cy.visit(path);
   }
 
+  static goToAndWaitForCounts(goTo: () => Cypress.Chainable) {
+    cy.intercept('GET', '/v1/counts').as('counts');
+
+    goTo();
+
+    cy.wait(['@counts'], { timeout: 10000 });
+  }
+
```

Comment:
@richard-cox Good call on making this commonly available. thanks for doing that! 
Unfortunately, we will still need this code to fix the flakiness:
```
 if (Cypress.env('username') === 'admin') {
       cy.wait(['@counts', '@counts'], { timeout: 10000 });
     } else {
       cy.wait('@counts');
     }
```
For context, the reason for the flakiness is Cypress interacts with the user menu button element before the page is done loading. The interaction occurs while api calls are being made in the background and the UI is updated once the calls are complete (ending with `counts`) which sometimes causes the user menu to close after being opened so adding the intercept to wait for the `counts` fixed the issue. However, the `counts` call occurs twice for admin users and only once for standard users when navigating to the home page which is why I wrapped the wait in an `if` statement. To avoid using the `if` statement, I've tried looking for ways to wait for all calls to complete before interacting with the page elements but I've had no luck there.

Reply:
oh nice! okay I'll address the conflict. thanks!

Reply:
Given those other changes we might be able to get away without the counts check, but it's nice to have some kind of 'page has made a request at the end of the generic init stuff' blocker